apps/framework-cli/src/cli/routines.rs

```diff
@@ -579,7 +589,14 @@ pub async fn start_production_mode(
     let infra_map: &'static InfrastructureMap = Box::leak(Box::new(plan.target_infra_map));
 
     web_server
-        .start(route_table, consumption_apis, infra_map, project, metrics)
+        .start(
+            route_table,
+            consumption_apis,
+            infra_map,
+            project,
+            metrics,
+            None,
```

Comment:
Why don't we provide the API doc in prod?

Reply:
if this is on the public webserver, we don't want that to be public do we?